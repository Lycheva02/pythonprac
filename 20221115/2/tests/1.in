class C:
    num = Num()

a = C()
print(a.num)
a.num = [1,2,3,4,5,6,7,8,9,10]
print(a.num)
a.num = 744
print(a.num)