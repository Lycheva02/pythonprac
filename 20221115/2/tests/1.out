0
10
744
